### ISSUE TITLE
Deprecated link found within the repository

### ISSUE THREAD

**Issue by nourbenamor2001** [jquense/react-big-calendar#2549]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://github.com/jquense/react-big-calendar/assets/146457416/c1d22ae8-302e-4f91-90a7-4f1a6838adb4

### Bug description

Within the Drag and Drop section, and the AddOns section:
![image](https://github.com/jquense/react-big-calendar/assets/146457416/5d9ba7f3-4880-4878-8cdb-a4062f15a919)

When I try to click on the link within Drag and Drop, I get this page error:
![image](https://github.com/jquense/react-big-calendar/assets/146457416/5db444b0-2ee3-4675-a936-ffb74b0c93c0)

https://github.com/jquense/react-big-calendar/assets/146457416/c1d22ae8-302e-4f91-90a7-4f1a6838adb4

### Expected Behavior

Expect to be redirected to Drag and Drop examples.

### Actual Behavior

https://github.com/jquense/react-big-calendar/assets/146457416/c1d22ae8-302e-4f91-90a7-4f1a6838adb4

### react-big-calendar version

1.10.0

### React version

16.14.0

### Platform/Target and Browser Versions

Windows 10

### Validations

- [X] Read the [docs](http://jquense.github.io/react-big-calendar/examples).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a react-big-calendar issue and not an implementation issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by forceddd** [comment]
@nourbenamor2001 The second link **AddOns** is available.
![image](https://github.com/jquense/react-big-calendar/assets/63057542/d2d0db0e-7f29-4bb8-8aa3-8d51f0a8e35f)

**Comment by nourbenamor2001** [comment]
> @nourbenamor2001 The second link **AddOns** is available. ![image](https://private-user-images.githubusercontent.com/63057542/318359551-d2d0db0e-7f29-4bb8-8aa3-8d51f0a8e35f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE5ODQ1MjksIm5iZiI6MTcxMTk4NDIyOSwicGF0aCI6Ii82MzA1NzU0Mi8zMTgzNTk1NTEtZDJkMGRiMGUtN2YyOS00YmI4LThhYTMtOGQ1MWYwYThlMzVmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDAxVDE1MTAyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ1MDFlOWRhMjU1ZTJlNWJiOTlmZGI3NzdmNmFiNGZlNzEyNGYxMzVhMmYwYzZiY2NjYmJlMjU4MWQ1NWNlNjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Z0_WVBDK5mj4kYpBHVS2gyYOdAnWhaYgkt5M2AkQ2hA)

Yes but the DnD link within the addOns page is also deprecated, it links to an error page.

**Comment by forceddd** [comment]
> > @nourbenamor2001 The second link **AddOns** is available. 
> 
> Yes but the DnD link within the addOns page is also deprecated, it links to an error page.
Are you referring to this page?
https://jquense.github.io/react-big-calendar/examples/index.html?path=/docs/addons-introduction--page

**Comment by nourbenamor2001** [comment]
> > > @nourbenamor2001 The second link **AddOns** is available.
> > 
> > 
> > Yes but the DnD link within the addOns page is also deprecated, it links to an error page.
> 
> Are you referring to this page? https://jquense.github.io/react-big-calendar/examples/index.html?path=/docs/addons-introduction--page

Yes when I clicked on the blue Drag and Drop sentence, it also led me to an error page, feel free to view the screen recording it might help you see what I get.

**Comment by forceddd** [comment]
I noticed that the version you are using is 1.10.0, you can try to upgrade the version, I am using the latest version and it works fine
